pyanaconda/modules/subscription/subscription_interface.py

```diff
@@ -171,3 +171,13 @@ def RegisterOrganizationKeyWithTask(self) -> ObjPath:
         return TaskContainer.to_object_path(
             self.implementation.register_organization_key_with_task()
         )
+
+    @emits_properties_changed
```

Comment:
I was wrong. The decorator is not necessary, because the properties won't be changed by the method, but by the task.

Reply:
No problem, dropping the decorator. :)